README.md

```diff
@@ -28,7 +28,7 @@ Simply replace `docker compose up -d <service>` with `docker rollout <service>`
 mkdir -p ~/.docker/cli-plugins
 
 # Download docker-rollout script to Docker cli plugins directory
-curl https://raw.githubusercontent.com/wowu/docker-rollout/master/docker-rollout -o ~/.docker/cli-plugins/docker-rollout
+curl https://raw.githubusercontent.com/wowu/docker-rollout/refs/heads/main/docker-rollout -o ~/.docker/cli-plugins/docker-rollout
```

Comment:
The current url works correctly, but if you want to update the main branch name, I suggest to drop the `refs/heads` part, as it's not required and makes the url shorter:

```suggestion
curl https://raw.githubusercontent.com/wowu/docker-rollout/main/docker-rollout -o ~/.docker/cli-plugins/docker-rollout
```